relay-general/src/store/normalize/span/tag_extraction.rs

```diff
@@ -125,9 +125,9 @@ fn extract_shared_tags(event: &Event) -> BTreeMap<SpanTagKey, String> {
             .and_then(|r| r.method.value())
             .map(|m| m.to_uppercase());
 
-        if let Some(transaction_method) = transaction_method_from_request
-            .or(http_method_from_transaction_name(transaction_name).map(|m| m.to_uppercase()))
-        {
+        if let Some(transaction_method) = transaction_method_from_request.or_else(|| {
```

Comment:
this suddenly made me `make lint-rust` fail, even though it got successfully merged in a previous PR. what's up with that?

Reply:
Not sure, but the change makes sense.